packages/gatsby/cache-dir/navigation.js

```diff
@@ -40,10 +40,12 @@ const onPreRouteUpdate = (location, prevLocation) => {
   }
 }
 
-const onRouteUpdate = (location, prevLocation) => {
+const onRouteUpdate = (location, prevLocation, announceLocation) => {
   if (!maybeRedirect(location.pathname)) {
     apiRunner(`onRouteUpdate`, { location, prevLocation })
-
+    if (announceLocation) {
+      announceLocation()
+    }
```

Comment:
Should this logic be moved to the `RouteUpdates` component?

My initial thought was that it may be easier to maintain if moved there, since its props will always update when the location changes, meaning if we use this API in other places then we wouldn't have to duplicate any logic

Though on second thoughts, it looks like the `onRouteUpdate` API is only called from the `RouteUpdates` component, so maybe it wouldn't make a difference either way?